csvlint.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = [">= 2.5", "< 3.4"]
+  spec.required_ruby_version = [">= 2.5", "< 3.5"]
```

Comment:
Please create another PR for this change.
It is out of scope.

Reply:
No need for another PR, there's already one open to make that change, but it's waiting on a bugfix for cucumber.

Reply:
I've merged that now, might be able to just rebase this.